bls/src/types/compressed_public_key.rs

```diff
@@ -19,15 +19,15 @@ use crate::PublicKey;
 /// and one bit indicating if it is the "point-at-infinity".
 #[derive(Clone)]
 pub struct CompressedPublicKey {
-    pub public_key: [u8; 285],
+    pub public_key: [u8; 95],
 }
 
 impl CompressedPublicKey {
-    pub const SIZE: usize = 285;
+    pub const SIZE: usize = 95;
```

Comment:
I haven't checked this number.

Reply:
It is correct. A single field element is 95 bytes, three field elements are 285 bytes.